WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -67,7 +67,7 @@ extension UIImage {
     static var cameraImage: UIImage {
         return UIImage.gridicon(.camera)
             .imageFlippedForRightToLeftLayoutDirection()
-            .applyTintColor(.placeholderImage)!
+            .withTintColor(.placeholderImage)
```

Comment:
💯 

While checking why would we be force-unwrapping this, I saw there's several instances of `imageWithTintColor(tintColor)!` as well, which has been renamed to `withTintColor(_:)` and the force-unwrapping can be removed too. Would make sense to fix these as well as part of this PR? Or I can open a new issue otherwise 👍

Reply:
Thanks for the hint here @iamgabrielma 
We have some references to the `WordPressUI` function `imageWithTintColor`, which returns an optional, hence the force unwrapping. I didn't change those, because I didn't know if by doing so we would get any unexpected change (`imageWithTintColor` has its own implementation that might differ from Apple's). Because of that, I think creating a new issue to check on those is the right course of action here.

Reply:
Gotcha! I opened a new one here: https://github.com/woocommerce/woocommerce-ios/issues/8151